src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -0,0 +1,178 @@
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *  
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *  
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program in the file lgpl21.txt
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.loklak.api.search;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.loklak.server.APIException;
+import org.loklak.server.APIHandler;
+import org.loklak.server.AbstractAPIHandler;
+import org.loklak.server.Authorization;
+import org.loklak.server.BaseUserRole;
+import org.loklak.server.Query;
+import org.loklak.susi.SusiThought;
+import org.loklak.tools.storage.JSONObjectWithDefault;
+
+public abstract class GithubProfileScraper extends AbstractAPIHandler implements APIHandler {
+
+	private static final long serialVersionUID = -4166800345379685201L;
+
+	@Override
+	public String getAPIPath() {
+		return "/api/githubprofilescraper.json";
+	}
+
+	@Override
+	public BaseUserRole getMinimalBaseUserRole() {
+		return BaseUserRole.ANONYMOUS;
+	}
+
+	@Override
+	public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights, JSONObjectWithDefault permissions)
+			throws APIException {
+		String profile = call.get("profile", "");
+		return scrapeGithub(profile);
+	}
+
+	public static SusiThought scrapeGithub(String profile) {
+
+		Document html = null;
+
+		JSONObject githubProfile = new JSONObject();
+
+		try {
+			html = Jsoup.connect("https://github.com/" + profile).get();
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+
+		String imageLink = html.getElementsByAttributeValue("property", "og:image").attr("content");
+		githubProfile.put("image_link", imageLink);
+
+		String type = html.getElementsByAttributeValue("property", "og:type").attr("content");
+		githubProfile.put("type", type);
+
+		String profileType = html.getElementsByAttributeValue("property", "og:type").attr("content");
+		githubProfile.put("type", profileType);
+
+		String username = html.getElementsByAttributeValue("property", "profile:username").attr("content");
+		githubProfile.put("username", username);
+
+		String description = html.getElementsByAttributeValue("property", "og:description").attr("content");
+		githubProfile.put("description", description);
+
```

Comment:
The OG tags and so on are used for SEO and not for showing content, scraping using these is not the right way to go about this and I think you need to scrape the actual HTML where the same content exists